chainstate/tx-verifier/Cargo.toml

```diff
@@ -17,6 +17,7 @@ pos_accounting = {path = '../../pos_accounting'}
 utils = {path = '../../utils'}
 utxo = {path = '../../utxo'}
 
+static_assertions.workspace = true
```

Comment:
This is nitpicky, but we're trying to follow a specfic way here.

workspace elements alone, versions alone, and crates from ".." alone. Each of those groups should be sorted by name.